users/vieting/experiments/switchboard/ctc/feat/network_helpers/perturbation.py

```diff
@@ -88,7 +97,7 @@ def sox(self, audio, sample_rate, random_state):
             speed = True
         if self._tempo is not None and random_state.random() < self._tempo.prob and not speed:
             factor = random_state.random() * (self._tempo.max - self._tempo.min) + self._tempo.min
-            tfm.tempo(factor)
+            tfm.stretch(factor)
```

Comment:
Did you verify this does exactly the same?

Reply:
yes, it does. the only difference is small performance gains depending on the factor value